app/src/androidTest/java/org/mozilla/fenix/ui/robots/SyncedTabsRobot.kt

```diff
@@ -39,6 +39,8 @@ private fun goBackButton() =
     onView(allOf(withContentDescription("Navigate up")))
 
 private fun assertSyncedTabsMenuHeader() {

```

Comment:
Nit: rename the test something like this so you don't need the inline comment: 
```suggestion
 private fun `GIVEN xxxx WHEN xxxx THEN xxxx`() {
```

Reply:
For UI tests, the convention doesn't seem to be 'GIVEN WHEN THEN' but rather the assert style based on a few other robots I looked at. Tagging @AaronMT or @rpappalax to give input

Reply:
that's totally fine, I didn't realize 😄